addons/payment_custom/views/payment_provider_views.xml

```diff
@@ -34,6 +34,18 @@
                             />
               </div>
             </field>
+            <page name="credentials" position="attributes">
+                <attribute name="invisible" add="custom_mode == 'wire_transfer'" separator=" or "/>
+            </page>
+            <field name="pre_msg" position="attributes">
+                <attribute name="invisible" add="custom_mode == 'wire_transfer'" separator=" or "/>
+            </field>
+            <field name="done_msg" position="attributes">
+                <attribute name="invisible" add="custom_mode == 'wire_transfer'" separator=" or "/>
+            </field>
+            <field name="cancel_msg" position="attributes">
+                <attribute name="invisible" add="custom_mode == 'wire_transfer'" separator=" or "/>
+            </field>
```

Comment:
Why add this at the end of the file? It'd be more logical to insert each xpath in between other existing xpaths in an order that follows the inherited template's nodes order.